### ISSUE TITLE
Natives platform of dependencies not changeable 

### ISSUE THREAD

**Issue by Protoxy22** [fabmax/kool-templates#1]
Hi, there is a problem with natives for others distributions than windows.

The dependencies of lwjgl search natives for windows (even with "natives-linux" on the build.gradle), it's maybe one of the dependencies that is searching it for windows.
And there is same problem with PhysX-jni we can't choose which native we want.

Thanks in advance
![image](https://github.com/fabmax/kool-templates/assets/30320751/fc5217c2-4af8-4549-949f-f6ad5e899899)

```
Caused by: java.lang.IllegalStateException: Failed loading native PhysX libraries for platform LINUX
	at de.fabmax.physxjni.Loader.load(Loader.java:25)
	at physx.NativeObject.<clinit>(NativeObject.java:7)
	... 7 more
Caused by: java.lang.ClassNotFoundException: de.fabmax.physxjni.linux.NativeLibLinux
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at de.fabmax.physxjni.Platform.getLib(Platform.java:17)
	at de.fabmax.physxjni.Loader.load(Loader.java:17)
	... 8 more
```

**Comment by Protoxy22** [comment]
There is also another bug, I can't use `Assets.launch {} `

Cannot access 'kotlinx.coroutines.CoroutineScope' which is a supertype of 'de.fabmax.kool.Assets'. Check your module classpath for missing or conflicting dependencies

I don't know if this error only occurs on my computer

**Comment by fabmax** [comment]
Ah you are right, the physx-jni native libs were missing. I [added](https://github.com/fabmax/kool-templates/commit/eac7abbc859ae884a1097a5cb9df8aacaa52e400) those and changed the buildscript to include all supported platforms by default. This doesn't hurt, only makes the final distribution a bit bigger.

For the coroutines error, this was also caused by a missing dependency (which I added now). I changed the corresponding dependency declaration in kool from `implementation` to `api`, so, with the next version, this shouldn't be needed anymore.
